src/Kunstmaan/MediaBundle/Validator/Constraints/HasGuessableExtensionValidator.php

```diff
@@ -68,6 +98,7 @@ public function setExtensionGuesser(ExtensionGuesserFactoryInterface $extensionG
      */
     public function setMimeTypeGuesser(MimeTypeGuesserFactoryInterface $mimeTypeGuesserFactory)
     {
+        @trigger_error('Calling the setMimeTypeGuesser method of this service is deprecated since KunstmaanMediaBundle 5.6 and will be replaced by the "@mime.types" service in KunstmaanMediaBundle 6.0.', E_USER_DEPRECATED);
```

Comment:
```suggestion
        @trigger_error('Calling the "setMimeTypeGuesser" method on "\Kunstmaan\MediaBundle\Validator\Constraints\ HasGuessableExtensionValidator" is deprecated since KunstmaanMediaBundle 5.6 and the method will be removed in KunstmaanMediaBundle 6.0. Inject the required services through the constructor instead.', E_USER_DEPRECATED);
```